src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IParameterReferenceExpression.cs

```diff
@@ -215,7 +215,7 @@ public void UsedParameterMethod1(Action a)
 ";
             string expectedOperationTree = @"
 IOperation:  (OperationKind.None) (Syntax: 'new Action( ... })')
-  Children(1): ILambdaExpression (Signature: lambda expression) (OperationKind.LambdaExpression, Type: System.Action) (Syntax: '() => ... }')
+  Children(1): ILambdaExpression (Signature: lambda expression) (OperationKind.LambdaExpression, Type: null) (Syntax: '() => ... }')
```

Comment:
I think this is one the scenarios that @AlekseyTs was talking about in the last design meeting, where VB will create an IConversionExpression here, but C# will not. This node is actually a BoundDelegateCreationExpression, which currently implements only IOperation. There actually isn't an IConversion expression here at all. In order to correctly make this work for both VB and C#, I think BoundDelegateCreation is going to have to synthesize an IConversionExpression to go between whatever API we create for it and the ILambdaExpression.